api/model/api.go

```diff
@@ -574,7 +574,7 @@ type APIDefinitionSpec struct {
 	// UseStandardAuth enables simple bearer token authentication
 	UseStandardAuth bool `json:"use_standard_auth,omitempty"`
 
-	// UseBasicAuth               bool                  `json:"use_basic_auth"`
+	UseBasicAuth bool `json:"use_basic_auth,omitempty"`
```

Comment:
Could you add a liner comment above the field? It will added as description of that field in CRD.

Reply:
Of course - done!